### PR TITLE
Fix contributor link #205

### PR DESCRIPTION
**Pull request by ry-v1**
Fix contributor link #205. Please check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds
...
     <td align="center"><a href="https://github.com/evayde"><img src="https://avatars1.githubusercontent.com/u/25255815?v=4" width="100px;" alt=""/><br /><sub><b>Enrico Gruner</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=evayde" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/umer2001"><img src="https://avatars2.githubusercontent.com/u/35496058?v=4" width="100px;" alt=""/><br /><sub><b>Muhammad Umer Farooq</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=umer2001" title="Code">💻</a></td>
-    <td align="center"><a href="http://shuvalov.info"><img src="https://avatars2.githubusercontent.com/u/1410106?v=4" width="100px;" alt=""/><br /><sub><b>Shuvalov Anton</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=a" title="Documentation">📖</a></td>
+    <td align="center"><a href="http://shuvalov.info"><img src="https://avatars2.githubusercontent.com/u/1410106?v=4" width="100px;" alt=""/><br /><sub><b>Shuvalov Anton</b></sub></a><br /><a href="https://github.com/fennecdjay/Gwion/commits?author=a-mere-peasant" title="Documentation">📖</a></td>
```
Comment (fennecdjay) [inline README.md:123]: `href` should be https://github.com/a-mere-peasant.
`img src` should be https://avatars3.githubusercontent.com/u/50142453?s=400&u=4844f21d4f48431d854307ecdf4d1f4d80977ab9&v=4
The bold text should be ... well I don't know, let's ask @a-mere-peasant 
I hope the notation is clear.

**Review by fennecdjay — COMMENTED** [review]
Thank you for taking care of this.

I think you forgot to update the links.
I not sure what to use as username, so I'm gonna ask.
Could you please update the links as stated in the review in the mean time?

**Comment by fennecdjay** [comment]
@a-mere-peasant, what should we use as username?
(see, I would not show `fennecdjay`, but `Jérémie Astor`).
Thank you.

**Comment by a-mere-peasant** [comment]
> @a-mere-peasant, what should we use as username?
> (see, I would not show `fennecdjay`, but `Jérémie Astor`).
> Thank you.

 Harsh Jain or just Harsh would be fine. Thanks :)

**Comment by ry-v1** [comment]
Fixed.

**Comment by fennecdjay** [comment]
That's nice, thanks everyone.

**Comment by fennecdjay** [comment]
@all-contributors please add @ry-v1 for documentation.

**Pull request merged**